test_scripts/Polices/user_consent_of_Policies/ATF_Device_identifier_creation_upon_connect_no_apps.lua

```diff
@@ -35,47 +32,34 @@ require('user_modules/AppTypes')
 --[[ Preconditions ]]
 commonFunctions:newTestCasesGroup("Preconditions")
 
-function Test:Precondition_Connect_device()
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:Test_Connect_device()
   commonTestCases:DelayedExp(2000)
   self:connectMobile()
   EXPECT_HMICALL("BasicCommunication.UpdateDeviceList",
-  {
-    deviceList = {
-      {
-        id = config.deviceMAC,
-        name = "127.0.0.1",
-        transportType = "WIFI",
-        isSDLAllowed = false
+    {
+      deviceList = {
+        {
+          id = config.deviceMAC,
+          name = "127.0.0.1",
+          transportType = "WIFI",
+          isSDLAllowed = false
+        }
       }
     }
-  }
-  ):Do(function(_,data)
-  self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
-  end)
-  :Times(AtLeast(1))
-end
+  )
+  :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
 
---[[ Test ]]
-commonFunctions:newTestCasesGroup("Test")
-function Test:Check_device_identifier_added_to_lpt()
-  local is_test_fail = true
-  local file = io.open(pts_json, "r")
-  local json_data = file:read("*all") -- may be abbreviated to "*a";
-  file:close()
-  local json = require("modules/json")
-  local data = json.decode(json_data)
-  local deviceIdentificatorInPTS = next(data.policy_table.device_data, nil)
-  if (deviceIdentificatorInPTS == config.deviceMAC) then
-    commonFunctions:userPrint(33, "device_identifier ".. deviceIdentificatorInPTS.. " section is created")
-    is_test_fail = false
-  end
-  if(is_test_fail == true) then
-    self:FailTestCase("Test is FAILED.")
-  end
+  --SDL snapshot will not be created until Device is consented through registered application
```

Comment:
@istoimenova, In this way expectation won't be verified: "SDL must add new <device identifier> section in "device_data" section"